test/link/section-union/align-conflict.out

```diff
@@ -1,4 +1,4 @@
-error: Section "conflicting alignment" is defined with conflicting 4-byte alignment (offset 0) and address $cafe
+error: Section "conflicting alignment" is defined with 4-byte alignment (offset 0) at section-union/align-conflict.asm(7), but with address $cafe at section-union/align-conflict.asm(7)
```

Comment:
These really are the same file; it's assembled once with `-DSECOND` and once without. Likewise for other `section-union/*.asm` tests.

Reply:
I feel like we should print the object file names so that this is less confusing. But we can also call this an edge enough case not to care.

Reply:
I'd prefer to leave them out: this seems like an edge case created mainly for the sake of testing, and usually the exact filestack+line number will be more useful than just the object file. (And showing both would be too verbose + usually redundant.)